pkg/remote/github/mock_GithubRepository_test.go

```diff
@@ -0,0 +1,33 @@
+// Code generated by mockery v2.3.0. DO NOT EDIT.
+
+package github
+
+import mock "github.com/stretchr/testify/mock"
+
+// MockGithubRepository is an autogenerated mock type for the GithubRepository type
+type MockGithubRepository struct {
+	mock.Mock
+}
```

Comment:
Let's have all our mocks in the same folder as to not be lost when we need to find existed mocks. For AWS we put them in `mocks` root folder. Either we continue like that and even create a subfolder `github`, or like you did we can put them inside the `remote/<provider>` folder.